balance.go

```diff
@@ -151,6 +159,10 @@ func (s *TransactionSource) UnmarshalJSON(data []byte) error {
 			json.Unmarshal(data, &s.Dispute)
 		case TransactionSourceFee:
 			json.Unmarshal(data, &s.Fee)
+		case TransactionSourceRecipientTransfer:
+			json.Unmarshal(data, &s.RecipientTransfer)
+		case TransactionSourcePayout:
+			json.Unmarshal(data, &s.Payout)
```

Comment:
Fixed! I apparently don't know my alphabet because I did check when I added that :p